Quizz App.postman_collection.json

```diff
@@ -0,0 +1,365 @@
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseurl}}/updatenote",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"updatenote"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Delete All Note",
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "{{baseurl}}//deleteallnote",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"",
+								"deleteallnote"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Questions",
+			"item": [
+				{
+					"name": "Insert Quiz",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"url": {
+							"raw": "{{baseurl}}/questions",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"questions"
+							]
+						},
+						"description": "The questions are updated directly by data.js to the database"
+					},
+					"response": []
+				},
+				{
+					"name": "Quiz Per Choice",
+					"protocolProfileBehavior": {
+						"disableBodyPruning": true
+					},
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"category\" : \"javascript\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseurl}}/question/category",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"question",
+								"category"
+							]
+						},
+						"description": "Get question per category (javascript,html,react,css ...)\n\nI want that the frontend should return the category choose for the quiz"
+					},
+					"response": []
+				},
+				{
+					"name": "Get all quiz",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{baseurl}}/questions",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"questions"
+							]
+						},
+						"description": "Get all the questions added in the databse"
+					},
+					"response": []
+				},
+				{
+					"name": "Delete all the quiz",
+					"request": {
+						"method": "DELETE",
+						"header": [],
+						"url": {
+							"raw": "{{baseurl}}/questions",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"questions"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Admin",
+			"item": [
+				{
+					"name": "Add Admin",
+					"request": {
+						"method": "PUT",
+						"header": [],
+						"url": {
+							"raw": "{{baseurl}}/admin",
+							"host": [
+								"{{baseurl}}"
+							],
+							"path": [
+								"admin"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		}
+	]
+}
```

Comment:
Not a big deal, but I didn't see any postman requests for the attempts. Did you intend to add any?

Reply:
attempts postman collection is in userProgress.postman_collection.json